adcm_aio_client/core/host_groups/_common.py

```diff
@@ -92,6 +92,15 @@ async def _request_endpoint(
 
         return await self._requester.get("hosts", query=query)
 
+    async def iter(self: Self, **filters: FilterValue) -> AsyncGenerator["Host", Any]:
+        response = await self._request_endpoint(query={}, filters=filters)
+        results = response.as_dict()["results"]
```

Comment:
It returns only the first page of hosts. This `iter` should work as `PaginatedAccessor.iter` since here we process response from paginated endpoint

<details>
  <summary>Tested with this code</summary>
    
  ```python
async def main3():
    async with ADCMSession(url="http://localhost:8000", credentials=Credentials("admin", "admin")) as adcm_client:
        for cluster in await adcm_client.clusters.all():
            await cluster.delete()

        for host in await adcm_client.hosts.all():
            await host.delete()

        cluster_bundle = await adcm_client.bundles.get(name__eq="sample_cluster")

        cl = await adcm_client.clusters.create(bundle=cluster_bundle, name="cl")
        chg = await cl.config_host_groups.create(name="chg")
        hp = await adcm_client.hostproviders.get(name__eq="hp")

        for i in range(60):
            await adcm_client.hosts.create(hostprovider=hp, name=f"host-{i+1}", cluster=cl)


        all_hosts = await adcm_client.hosts.all()
        assert len({h.id for h in all_hosts}) == 60

        await chg.hosts.add(host=all_hosts)

        cgh_hosts = await chg.hosts.all()
        len_chg_hosts = len({h.id for h in cgh_hosts})
        assert len_chg_hosts == 60, f"{len_chg_hosts} != 60"
  ```
</details>

Reply:
Also, maybe we need to specify ordering by `id` in `self._request_endpoint` to be sure, that paging works correctly

Reply:
@DanBalalan maybe `limit=len(ids)` should be passed in `_request_endpoint`? we don't want to imitate paging where there's none of it

Reply:
@Sealwing yes, thats better

Reply:
added limit=len(ids)

Reply:
`ids` here is a string in format `"<id>,<id>,..."`. It will work because length of such string is greater than length of ids collection itself. But i suggest make something like:
```
ids = {host["id"]) for host in data}
limit = len(ids)
ids = ",".join(str(id_) for id_ in ids)
```

Reply:
changed